dispatch/static/manager/src/js/components/ZoneEditor/index.js

```diff
@@ -30,8 +30,33 @@ class ZoneEditorComponent extends React.Component {
     )
   }
 
+  processZone(zone) {
+    if (!zone) {
+      return zone
+    }
+
+    function processWidget(widget, fields) {
```

Comment:
Just to clarify, `processWidget` just removes the 'widget' property from each nested widget, right?